imports/ui/pages/layout.jsx

```diff
@@ -28,11 +28,11 @@ const Layout = React.createClass({
       <div className='full-height'>
         <SubscribeProjects />
         <Navbar />
-        <div className='content m-b-1'>
+        <Loader top='32px' left='calc(100% - 35px)' />
```

Comment:
Don't really understand the props here.  If the parent is indeed full-height, won't the loader be centered anyway?